src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -3540,23 +3540,23 @@ s32 func_80836AB8(Player* this, s32 arg1) {
     return var;
 }
 
-// Update things related to Z Targeting
+// Player_UpdateZTargeting
 void func_80836BEC(Player* this, PlayState* play) {
     s32 ignoreLeash = false;
-    s32 zTrigPressed = CHECK_BTN_ALL(sControlInput->cur.button, BTN_Z);
-    Actor* actorToTarget;
+    s32 zButtonHeld = CHECK_BTN_ALL(sControlInput->cur.button, BTN_Z);
+    Actor* nextLockOnActor;
     s32 pad;
-    s32 holdTarget;
-    s32 cond;
+    s32 isHoldTargeting;
```

Comment:
`useHoldTargeting`? "hold targeting" does not seem very adjective-y to me but idk

Reply:
its intended to be like "is hold targeting the current setting?"
if we said "use" id want it to be "using hold targeting" but that makes it sound like something you can change at will even during gameplay. idk as well

Reply:
eh with more thought `usingHoldTargeting` seems fine